src/pages/components/feature-section.mdx

```diff
@@ -28,7 +28,7 @@ Additional usage guidelines will be available soon.
 
 ## Overview
 
-Feature section is a full-width section, designed to create a visual break from the rest of the page and focus the viewer's attention to one single story. It contains expressive imagery and a CTA linking people to more details.
+Feature section is a full-width section designed to create a visual break from the rest of the page and focus the viewer's attention on a single story. It contains expressive imagery and a CTA linking people to more details. Feature section is meant to be used sparingly to draw attention – avoid stacking multiple Feature sections in a row or including too many on a single page.
```

Comment:
Likewise here, the second `Feature section` is capitalized too.

Reply:
aw man good catch, we are trying not to capitalize any component names unless they are in the middle of the sentence, but we keep messing that up lol